chains/solana/contracts/programs/ccip-router/src/event.rs

```diff
@@ -148,3 +128,27 @@ pub struct OwnershipTransferred {
     pub from: Pubkey,
     pub to: Pubkey,
 }
+
+/////////////////////////////////
+// Token Admin Registry Events //
+/////////////////////////////////
```

Comment:
Nit: This is a nit that I had for tobi as well: I personally don't like "header comments" like this, particularly in Rust, as they're a poor substitute for inline modules. It's easy for the comment to go out of sync with the code, and for the block covered by the comment to become unclear.

Instead, I'd prefer `pub mod token_admin_registry`, which makes it clearer when importing (i.e. events::token_admin_registry::PoolSet`.

Reply:
Good point here, using modules as you did for seeds makes everything more clear. I need to figure out if Anchor auto gen code will work the same way anywas 🤔

Reply:
Worked! 🙌

Reply:
I refactored all events